imagersite/imager_profile/models.py

```diff
@@ -0,0 +1,92 @@
+"""Define imager profile class and it's methods."""
+
+
+from django.db import models
+from django.db.models.signals import post_save
+from django.contrib.auth.models import User
+from django.dispatch import receiver
+from django.utils.encoding import python_2_unicode_compatible
+
+# Create your models here.
+
+
+class ImagerActiveManager(models.Manager):
+    """Manager for the ImagerProfile Active API."""
+
+    def get_queryset(self):
+        """Return query of active users."""
+        return super(ImagerActiveManager, self).get_queryset().filter(is_active=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    """A profile for users to our application."""
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    location = models.CharField(max_length=255)
+    age = models.IntegerField()
+    camera_type = models.CharField(max_length=255)
+    job = models.CharField(max_length=255)
+    url = models.URLField(max_length=200)
+    objects = models.Manager()
+    active = ImagerActiveManager()
```

Comment:
consider adding `blank=True` for text field and `null=True` for integers so that your users don't have to fill out every single field.